R/register_ctx.R

```diff
@@ -54,72 +54,72 @@
 #'   \code{NA_character_}.
 #'
 #'  * `has_ctx_key()` returns a Boolean.
-#' @name register_ccdr
+#' @name register_ctxR
 
 
 
-#' @rdname register_ccdr
+#' @rdname register_ctxR
 #' @export
 #' @examplesIf has_ctx_key() & is.na(ctx_key() == 'FAKE_KEY')
 #' # Check if API key is showing
 #' showing_key()
 
 showing_key <- function() {
 
-  if (has_ccdr_option('display_api_key')) {
-    getOption('ccdr')$display_api_key
+  if (has_ctxR_option('display_api_key')) {
+    getOption('ctxR')$display_api_key
   } else {
     FALSE
   }
 }
 
 
 
-#' @rdname register_ccdr
+#' @rdname register_ctxR
 #' @export
 #' @examplesIf has_ctx_key() & is.na(ctx_key() == 'FAKE_KEY')
 #' # Toggle API key to display
-#' ccdr_show_api_key()
+#' ctxR_show_api_key()
 
-ccdr_show_api_key <- function() {
-  set_ccdr_option('display_api_key' = TRUE)
-  cli::cli_alert_warning('ccdR will now display PRIVATE API keys in the console.')
+ctxR_show_api_key <- function() {
+  set_ctxR_option('display_api_key' = TRUE)
+  cli::cli_alert_warning('ctxR will now display PRIVATE API keys in the console.')
   invisible()
 }
 
 
 
-#' @rdname register_ccdr
+#' @rdname register_ctxR
 #' @export
 #' @examplesIf has_ctx_key() & is.na(ctx_key() == 'FAKE_KEY')
 #' # Toggle API key to be hidden
-#' ccdr_hide_api_key()
+#' ctxR_hide_api_key()
 
-ccdr_hide_api_key <- function() {
-  set_ccdr_option('display_api_key' = FALSE)
-  cli::cli_alert_info('ccdR will now suppress private API keys in the console.')
+ctxR_hide_api_key <- function() {
+  set_ctxR_option('display_api_key' = FALSE)
+  cli::cli_alert_info('ctxR will now suppress private API keys in the console.')
   invisible()
 }
 
 
 
-#' @rdname register_ccdr
+#' @rdname register_ctxR
 #' @export
 #' @examplesIf has_ctx_key() & is.na(ctx_key() == 'FAKE_KEY')
 #' # Register key for this session
-#' register_ccdr(key = 'YOUR API KEY')
+#' register_ctxR(key = 'YOUR API KEY')
 #' # Register key over sessions
-#' register_ccdr(key = 'YOUR API KEY', write = TRUE)
+#' register_ctxR(key = 'YOUR API KEY', write = TRUE)
 
-register_ccdr <- function(key, write = FALSE) {
+register_ctxR <- function(key, write = FALSE) {
```

Comment:
Perhaps we should rename this function as "register_ctx_api_key" ? Or could model after tcpl like "ctxR_conf" (for "Configure")

Update line 1 as "Register CTX API Key for ctxR" or "Configure environment with CTX API key"

Reply:
I would opt for the "register_ctx_api_key" and "Register CTX API Key for ctxR" pair.